plugins/modules/packaging/os/redhat_subscription.py

```diff
@@ -814,8 +814,6 @@ def main():
         mutually_exclusive=[['activationkey', 'username'],
                             ['activationkey', 'consumer_id'],
                             ['activationkey', 'environment'],
-                            ['activationkey', 'autosubscribe'],
```

Comment:
Why is it OK to remove this one? `autosubscribe` is an alias of `auto_attach`, so maybe that one should be specified?

Reply:
ok, it seems it's a mistake but therein lies another doubt.

I removed it because there was no parameter named `autosubscribe`, but then now, double checking on it, I realise it is an alias for `auto_attach`. So the next question is: does `mutually_exclusive` works with aliases as well? Even if it does, I think we should try and use the _canonical_  name, so I am reverting this but changing it to `auto_attach`.

Reply:
in fact, I get a:
```
ERROR: plugins/modules/packaging/os/redhat_subscription.py:0:0: mutually_exclusive-unknown: mutually_exclusive contains terms which are not part of argument_spec: autosubscribe
```
with the alias, that's what prompted me to (inadvertently) remove that line before checking.

Reply:
It does work with the alias, but not in the way people expect: it prevents that the alias is used with `activationkey`, but you can still use `auto_attach` with `actionkey`.

With your current change, this is no longer possible, so it's a potentially breaking change.